src/sentry/api/serializers/rest_framework/notification_action.py

```diff
@@ -272,6 +278,7 @@ def validate_discord_channel(
             )
 
         try:
+            assert channel_id, "Channel id must exist to validate channel id"
```

Comment:
actually I think this points out a logic error on line 275 -- I don't think the second half of that if makes any sense?

Reply:
wait yeah, and the docstring says that both `channel_id` & `channel_name` need to exist but we don't use `channel_name` either whaaa

Reply:
I bet we persist the channel name so it can be used in the UI, and want it to be a required field.